### PR TITLE
nodejs-fr: bootstrap get-involved page

### PR DESCRIPTION
**Pull request by clakech**

### CODE REVIEW

File: locale/fr/get-involved/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Get involved
```
Comment (Tiriel) [inline locale/fr/get-involved/index.md]: Do we let the title in english? Maybe translate it too?

File: locale/fr/get-involved/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant 
+d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
+[reporter un problème](https://github.com/nodejs/node/issues), aider à la documentation
```
Comment (Tiriel) [inline locale/fr/get-involved/index.md]: Maybe "signaler" in stead of "reporter", which seems to be an anglicism

File: locale/fr/get-involved/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ou contribuer au code du projet, vous êtes au bon endroit. Explorez les ressources 
+de notre communauté pour trouver comment vous pouvez aider:
+
+<hr>
+
+## Discussion
+
+- La [list des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités coeurs de Node.js.
```
Comment (Tiriel) [inline locale/fr/get-involved/index.md]: Missing "e" in "liste"

File: locale/fr/get-involved/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+- Pour une discussion temps réelle à propos de développement Node, allez sur `irc.freenode.net` dans le canal `#node.js` avec un [client IRC](https://fr.wikipedia.org/wiki/Liste_de_clients_IRC) ou connectez vous depuis votre navigateur web sur le canal en utilisant le canal [freenode's WebChat](http://webchat.freenode.net/?channels=node.js).
```
Comment (Tiriel) [inline locale/fr/get-involved/index.md]: I think it should be "temps réel", masculine, and "*du* développement Node".
And "canal" is repeated too many times in the last part. Maybe "sur le canal en utilisant le [WebChat freenode]"?

**Review by Tiriel — COMMENTED** [review]
Reviewed some wordings, but I may have forgottent some typos. Great job anyway!

**Comment by clakech** [comment]
@Tiriel thx, PR updated following your review

**Review by Tiriel — APPROVED** [review]
Seems ok to me now!

File: locale/fr/get-involved/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant
```
Comment (vdeturckheim) [inline locale/fr/get-involved/index.md]: 'excitée de permettre' -> I'm not a big fan here.
* 'excitée a l'idee de permettre' ?
* something else ?

File: locale/fr/get-involved/index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant 
+d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
+[signaler un problème](https://github.com/nodejs/node/issues), aider à la documentation
```
Comment (vdeturckheim) [inline locale/fr/get-involved/index.md]: 'aider a ecrire/maintenir la documentation' ?

File: locale/fr/get-involved/index.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté
```
Comment (vdeturckheim) [inline locale/fr/get-involved/index.md]: Globally, this paragraph does not feel very idiomatic in French to me.

**Review by vdeturckheim — CHANGES_REQUESTED** [review]

File: locale/fr/get-involved/index.md (hunk 5 above)
Comment (clakech) [reply locale/fr/get-involved/index.md]: 'excitée a l'idee de permettre' => c'est tout much AMA

File: locale/fr/get-involved/index.md (hunk 6 above)
Comment (clakech) [reply locale/fr/get-involved/index.md]: aider à documenter ?

File: locale/fr/get-involved/index.md (hunk 7 above)
Comment (clakech) [reply locale/fr/get-involved/index.md]: delete 'les' ?

File: locale/fr/get-involved/index.md (hunk 5 above)
Comment (vdeturckheim) [reply locale/fr/get-involved/index.md]: yes, the problem is in the word "excite" actually. It's a too literal translation,

File: locale/fr/get-involved/index.md (hunk 6 above)
Comment (vdeturckheim) [reply locale/fr/get-involved/index.md]: Maybe, the way to express needs to be balanced in the whole paragraph.

File: locale/fr/get-involved/index.md (hunk 7 above)
Comment (Tiriel) [reply locale/fr/get-involved/index.md]: I'd say if you want to delete "les", go singular. "Nous encourageons toute forme de contribution venant de la communauté" ?

File: locale/fr/get-involved/index.md (hunk 5 above)
Comment (Tiriel) [reply locale/fr/get-involved/index.md]: Maybe simply "heureuse"?

File: locale/fr/get-involved/index.md (hunk 7 above)
Comment (vdeturckheim) [reply locale/fr/get-involved/index.md]: +1

File: locale/fr/get-involved/index.md (hunk 5 above)
Comment (vdeturckheim) [reply locale/fr/get-involved/index.md]: better yes!

File: locale/fr/get-involved/index.md (hunk 6 above)
Comment (clakech) [reply locale/fr/get-involved/index.md]: I don't understand your latest comment

**Comment by clakech** [comment]
@vdeturckheim PR updated !

File: locale/fr/get-involved/index.md (hunk 6 above)
Comment (vdeturckheim) [reply locale/fr/get-involved/index.md]: At the end, reading this paragraph should feel like it was originally written in French.

**Review by vdeturckheim — APPROVED** [review]
lgtm

**Review by Tiriel — APPROVED** [review]
+1

**Comment by lpinca** [comment]
Thank you.

**Pull request merged**